src/utils.py

```diff
@@ -98,40 +97,21 @@ def get_mask(raster_img, vector_data, nan_value=0):
     return binary_mask
 
 
-def slice_image(img, size=(512, 512), overlap=6):
-    """Given an image slice according to size with no overlapping.
+def slice_image(img, size=(512, 512, 12), overlap=6):
```

Comment:
is 12 the channels? if so, then the calls of this functions has to be altered for 1 channel images. 
Does everything run with this edit?

Reply:
Well, I guess it wouldn't be hard to change to read the number of channels in img.

I'm able to slice img's read from rasterio, I haven't run the whole pipeline with it though.

Reply:
It won't be hard, but it's better to verify everything run before merging.